packages/devtools_app/lib/src/flutter/preferences.dart

```diff
@@ -22,26 +24,20 @@ class PreferencesController {
   ValueListenable get darkModeTheme => _darkModeTheme;
 
   void _init() {
-    // TODO(devoncarew): Enable when we have storage backed settings.
-    //if (storage == null) {
-    //  // This can happen when running tests.
-    //  log('PreferencesController: storage not initialized');
-    //  return;
-    //}
-
-    // TODO(devoncarew): Enable when we have storage backed settings.
+    if (storage == null) {
+      // This can happen when running tests.
+      log('PreferencesController: storage not initialized');
+      return;
+    }
+
     // Get the current values and listen for and write back changes.
-    //storage.getValue('ui.darkMode').then((String value) {
-    //  darkModeTheme.value = value == null || value == 'true';
-    //  darkModeTheme.addListener(() {
-    //    setTheme(darkTheme: darkModeTheme.value);
-    //    storage.setValue('ui.darkMode', '${darkModeTheme.value}');
-    //  });
-    //});
-
-    _darkModeTheme.addListener(() {
-      // ignore: deprecated_member_use_from_same_package
-      devtools_theme.setDarkTheme(_darkModeTheme.value);
+    storage.getValue('ui.darkMode').then((String value) {
+      _darkModeTheme.value = value == null || value == 'true';
```

Comment:
Is this intended to default to Dark mode if there's no stored value? (Other code, like in framework_core appears to default to light it `theme` isn't on the querystring).

Reply:
we might as well also change the default to the dark theme everywhere. Initially the default was the light theme in DevTools because that is what we worked on first and the light theme was a bit more polished in the dart:html version of the app. the dark theme in the flutter version of the app is about on par with the light theme.

Reply:
yup, the code above will default to dark mode (sounds like this is reasonable at this point)